corehq/apps/toggle_ui/templates/toggle/flags.html

```diff
@@ -27,6 +27,10 @@
       {% trans "Show usage metrics" %}
     </a>
   {% endif %}
+  <button id="export-button" data-bind="click: downloadFile" class="ko-template btn btn-default btn-xs pull-right">
+    <i class="fa fa-cloud-download"></i>
+    {% trans "Export to .xlsx" %}
```

Comment:
@snopoke I thought about that initially, but changed my mind because I like the fact that it's being explicit as to what format it will be exported in. If we'd include other export types also it would make sense to omit the format on the button...but this is only my preference...if you think we should omit the type I'd be happy to change it?

Reply:
Not a big deal. I just don't think it helps the UI at all. Since there is only one format the user doesn't have a choice and knowing the format beforehand won't change their decision about downloading it. They will also find out the format as soon as they download it.

I do notice however that in the reports UI we say `Export to Excel` so we should probably stick with that to keep it consistent.